.gitignore

```diff
@@ -1,3 +0,0 @@
-node_modules
-coverage
-output
```

Comment:
Let those lines, they are to ignore in the development workflow of the project
Just don't add files related to IDE here, put them locally (we don't need to share them)